tests/NimbleClientbase.py

```diff
@@ -0,0 +1,165 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
```

Comment:
Can we please run all of this code through pylint, pep-8, and/or pythonTidy?  I see many areas of inconsistency.  Some examples include extra spaces, no deterministic length of any strings, no consistent naming of variables (camelCase, PascalCase, etc.).  More time is spent reading code than writing code.  We should strive to make it as readable as possible and consistent throughout.

Reply:
sorry about that.. i was hoping that visual studio code already has pylint extension.. i checked it does. but perhaps not that smart enough...
please hold your review for further files.
Just review any one test case and i can follow the same every where.

also, i did not get "no deterministic length of any string".. ex?

Reply:
Use `flake8` instead. You can install the VSCode Python module and configure it to do that for you.